up-l1/README.adoc

```diff
@@ -261,6 +261,39 @@ end
 deactivate T
 ----
 
+In RegisterListener API, we only have the `resource_id`` of the source and sink.
+However, sometimes it's necessary to distinguish the message types which should be listened to.
+
+Here is the table to map the various combination of source `resource_id` and sink `resource_id` to different message types.
+
+[%autowidth]
+|===
+| src resource_id | sink resource_id | Publish | Notification | Request | Response
+
+| [8000-FFFE] |   None   |    V    |              |         |          
+| [8000-FFFE] |     0    |         |      V       |         |          
+|      0      | [1-7FFF] |         |              |    V    |          
```

Comment:
I don't think this is a valid usecase for source and sink matching

Reply:
I left the comments below. Please let me know your thoughts.